pallene/driver.lua

```diff
@@ -65,9 +70,11 @@ function driver.compile_internal(filename, input, stop_after)
         return module, errs
     end
 
-    module, errs = constant_propagation.run(module)
-    if stop_after == "constant_propagation" or not module then
-        return module, errs
+    if opt_level ~= "0" then
```

Comment:
I think it would be nice if we could use `opt_level > 0` here. What do you think about removing -Os, -Ofast and -Og, allowing only -O0, -O1, -O2, and -O3? If the programmer really wants those other optimization flags they could still use CFLAGS.

Reply:
I believe you are right. Since there is the option to override the C compiler optimization level with `CFLAGS`, -Os, fast, and g are really not needed. Already changed this. Now argparse ensures that the `-O` option only accepts numbers 0, 1, 2, and 3.